Infrastructure/arc_lambda/index.js

```diff
@@ -43,21 +45,82 @@ exports.handler = async(event, context, callback) => {
       output.on('end', function() {
         console.log('Data has been drained');
       });
-      
+      output.on('error', function (err) { 
+        console.log('[OUTPUT STREAM ERROR]', err); 
```

Comment:
can multiple output events be dispatched? what else can we do with the error other than logging to stderr? should we exit the function?

Reply:
This is lambda function, so if we can create archive, we'll add all error logs into that errors.txt; if we fail to create archive then we'll check CloudWatch. This operation (create archive) happens once when intake is closed (invoked by the cron job), so I don't think we need to complicate the function by adding push to SNS to send email etc.